packages/wrangler/src/type-generation/index.ts

```diff
@@ -262,11 +247,14 @@ async function generateTypes(
 		);
 	}
 
-	const envTypeStructure: string[] = [];
+	const envTypeStructure: [string, string][] = [];
```

Comment:
I think it would be nicer/clearer to have this as
```ts
const envTypeStructure: {
 key: string,
 type: string,
}[] = [];
```

but it would make all the `.push` calls below much more verbose, and the current code should be clear enough

but I am happy with either solution, if anyone prefers the more verbose version (or has any other idea) I'm happy to change this